docusaurus.config.js

```diff
@@ -27,6 +27,13 @@ const docusaurusConfig = {
         },
     ],
     themeConfig: {
+        announcementBar: {
+            id: 'site_announcement', // Any value that will identify this message.
+            content: `<a target="_blank" rel="noopener noreferrer" href="https://saucelabs.com/resources/community/slack">\n + Have a question? Join our <img alt="slack.png" src="/img/slack.png" width="17" height="17" /> Slack Community Workspace!</a>`,
```

Comment:
Suggestion to remove the "+" and move the Slack icon to the beginning of the paragraph. We could also change the text to say "Join us on SLack". 

```suggestion
            content: `<a target="_blank" rel="noopener noreferrer" href="https://saucelabs.com/resources/community/slack"><img alt="slack.png" src="/img/slack.png" width="17" height="17" /> Have a question? Join us on Slack!</a>`,
```
![Screenshot 2023-05-19 at 1 30 59 PM](https://github.com/saucelabs/sauce-docs/assets/15352964/1f566903-43ca-447e-89c9-5d96fe80b9c7)